Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -55,7 +49,7 @@ when the module lives in a lower universe.
 -/
 
 
-universe u v w
+universe v' u v w
```

Comment:
maybe `universe u v v' w` makes more sense

Reply:
I put `v'` at the first since I want it to be the first universe specified in the theorem. This might make it easier to use, similar to `[Small.{v} R]`. Is there a better way to do this?

Reply:
This makes sense!